src/ConnectedMode/UI/ManageConnections/ManageConnectionsDialog.xaml

```diff
@@ -80,8 +80,8 @@
                                    <RowDefinition Height="*"/>
                                </Grid.RowDefinitions>
 
-                               <TextBlock Grid.Row="0" Text="{Binding Path=Name, Mode=OneWay}" VerticalAlignment="Center" />
-                               <CheckBox Grid.Row="1" Content="{x:Static res:UiResources.SmartNotificationsCheckboxLabel}" IsChecked="{Binding Path=EnableSmartNotifications}"
+                               <TextBlock Grid.Row="0" Grid.RowSpan="2" Text="{Binding Path=Name, Mode=OneWay}" VerticalAlignment="Center" />
```

Comment:
Now there is a lot of space. Let's remove the top margin from line 56 
And maybe decrease the icon's width and height to 40 on line 63.

Reply:
The checkbox on the second row is temporarily hidden, and we're planning to bring it back in the upcoming sprint. Given that we also have further UI improvements planned, I’d suggest holding off on this particular change for now, unless it's a major UI concern.